### DISCUSSION TITLE
请问tarojs如何调用微信小程序的weappOrderConfirm确认收货呢？

### DISCUSSION THREAD

**Discussion by GuiltyKirito** [NervJS/taro#15305] [Q & A]
在实际业务过程中，需要用户触发微信小程序的确认收货功能，可能需要设置Taro.openBusinessView 的 businessType 为：weappOrderConfirm，但是在tarojs的官方文档里并未看到，请问各位有过业务经验的前辈是怎么实现的呢？

**Comment by TheKonka** [comment]
直接通过 wx.openBusinessView 调用就可以了

**Reply by GuiltyKirito** [reply]
原来是用这种方式来解决的吗？太厉害了，感谢~

**Reply by xileliu** [reply]
直接使用wx.openBusinessView taro中报语法错误怎么解决

**Reply by TheKonka** [reply]
> 直接使用wx.openBusinessView taro中报语法错误怎么解决

类型报错吧，简单粗暴的方法： `declare const wx: any`
